ui/raidboss/data/manifest.txt

```diff
@@ -1,266 +1,268 @@
-04-sb/trial/byakko-ex.txt
-04-sb/trial/lakshmi-ex.js
-04-sb/trial/lakshmi-ex.txt
-04-sb/trial/rathalos-ex.js
-04-sb/trial/seiryu-ex.js
-04-sb/trial/seiryu-ex.txt
-04-sb/trial/shinryu.js
-04-sb/trial/shinryu.txt
-04-sb/trial/shinryu-ex.js
-04-sb/trial/shinryu-ex.txt
-04-sb/trial/susano-ex.js
-04-sb/trial/susano-ex.txt
-04-sb/trial/suzaku-ex.js
-04-sb/trial/suzaku-ex.txt
-04-sb/trial/tsukuyomi-ex.js
-04-sb/trial/tsukuyomi-ex.txt
-04-sb/trial/yojimbo.js
-04-sb/trial/yojimbo.txt
-04-sb/ultimate/ultima_weapon_ultimate.js
-04-sb/ultimate/ultima_weapon_ultimate.txt
-04-sb/ultimate/unending_coil_ultimate.js
-04-sb/ultimate/unending_coil_ultimate.txt
-05-shb/alliance/the_copied_factory.js
-05-shb/alliance/the_copied_factory.txt
-05-shb/alliance/the_puppets_bunker.js
-05-shb/alliance/the_puppets_bunker.txt
-05-shb/dungeon/akadaemia_anyder.js
-05-shb/dungeon/akadaemia_anyder.txt
-05-shb/dungeon/amaurot.js
-05-shb/dungeon/amaurot.txt
-05-shb/dungeon/anamnesis_anyder.js
-05-shb/dungeon/anamnesis_anyder.txt
-05-shb/dungeon/dohn_mheg.js
-05-shb/dungeon/dohn_mheg.txt
-05-shb/dungeon/heroes_gauntlet.js
-05-shb/dungeon/heroes_gauntlet.txt
-05-shb/dungeon/holminster_switch.js
-05-shb/dungeon/holminster_switch.txt
-05-shb/dungeon/malikahs_well.js
-05-shb/dungeon/malikahs_well.txt
-05-shb/dungeon/mt_gulg.js
-05-shb/dungeon/mt_gulg.txt
-05-shb/dungeon/qitana_ravel.js
-05-shb/dungeon/qitana_ravel.txt
-05-shb/dungeon/the_grand_cosmos.js
-05-shb/dungeon/the_grand_cosmos.txt
-05-shb/dungeon/twinning.js
-05-shb/dungeon/twinning.txt
-05-shb/etc/the_diadem.js
-05-shb/raid/e1n.js
-05-shb/raid/e1n.txt
-05-shb/raid/e1s.js
-05-shb/raid/e1s.txt
-05-shb/raid/e2n.js
-05-shb/raid/e2n.txt
-05-shb/raid/e2s.js
-05-shb/raid/e2s.txt
-05-shb/raid/e3n.js
-05-shb/raid/e3n.txt
-05-shb/raid/e3s.js
-05-shb/raid/e3s.txt
-05-shb/raid/e4n.js
-05-shb/raid/e4n.txt
-05-shb/raid/e4s.js
-05-shb/raid/e4s.txt
-05-shb/raid/e5n.js
-05-shb/raid/e5n.txt
-05-shb/raid/e5s.js
-05-shb/raid/e5s.txt
-05-shb/raid/e6n.js
-05-shb/raid/e6n.txt
-05-shb/raid/e6s.js
-05-shb/raid/e6s.txt
-05-shb/raid/e7n.js
-05-shb/raid/e7n.txt
-05-shb/raid/e7s.js
-05-shb/raid/e7s.txt
-05-shb/raid/e8n.js
-05-shb/raid/e8n.txt
-05-shb/raid/e8s.js
-05-shb/raid/e8s.txt
-05-shb/trial/hades.js
-05-shb/trial/hades.txt
-05-shb/trial/hades-ex.js
-05-shb/trial/hades-ex.txt
-05-shb/trial/innocence.js
-05-shb/trial/innocence.txt
-05-shb/trial/innocence-ex.js
-05-shb/trial/innocence-ex.txt
-05-shb/trial/ruby_weapon.js
-05-shb/trial/ruby_weapon.txt
-05-shb/trial/ruby_weapon-ex.js
-05-shb/trial/ruby_weapon-ex.txt
-05-shb/trial/shiva-un.js
-05-shb/trial/shiva-un.txt
-05-shb/trial/titania.js
-05-shb/trial/titania.txt
-05-shb/trial/titania-ex.js
-05-shb/trial/titania-ex.txt
-05-shb/trial/varis-ex.js
-05-shb/trial/varis-ex.txt
-05-shb/trial/wol.js
-05-shb/trial/wol.txt
-05-shb/trial/wol-ex.js
-05-shb/trial/wol-ex.txt
-05-shb/ultimate/the_epic_of_alexander.js
-05-shb/ultimate/the_epic_of_alexander.txt
+00-misc/general.js
+00-misc/test.js
+00-misc/test.txt

```

Comment:
Hmm.  What's going on with this file? Did you change the line endings?

Reply:
I accidentally uploaded my local copy as a new file instead of just editing the existing file

Reply:
I accidentally uploaded my local copy as a new file instead of just editing the existing file